helpers/cli_version_check/cli_version/installed_cli_version.go

```diff
@@ -0,0 +1,42 @@
+package cli_version
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"regexp"
+)
+
+type InstalledCliVersion interface {
+	GetVersion() string
+	SetFullVersionString(string)
+}
+
+type cliVersion struct {
+	fullVersionStr string
+}
+
+func NewInstalledCliVersion() InstalledCliVersion {
+	output, err := exec.Command("cf", "-v").Output()
+	if err != nil {
+		fmt.Println("Error trying to determine CF CLI version:", err)
+		os.Exit(1)
```

Comment:
Rather than exiting, this should return an error that the upstream test asserts should not have occurred.